### DISCUSSION TITLE
Derivative Function to extract image URL issue

### DISCUSSION THREAD

**Discussion by Ken-JIAnomads** [AntlerVC/firetable#1250] [Support / Q&A] [answered]
I followed the Rowy Documentation and script. I named column WorkURL (key: workUrl)

![Screenshot 2023-05-09 at 10 20 00 pm](https://github.com/rowyio/rowy/assets/130031825/567a52cd-2566-469f-be53-d06eea44fad4)

From the Cloud Logs there appears to be an Type Error somewhere?  Help appreciated!

![Screenshot 2023-05-09 at 10 25 00 pm](https://github.com/rowyio/rowy/assets/130031825/400964ae-2d9c-4ae2-9bb6-0715ace029fa)

**Comment by rodgetech** [comment, marked as answer]
I think this is what you want:

```typescript
let url = row.workTypeImage[0].downloadURL
```
Assuming the key of your WorkTypeImage column is workTypeImage.

**Reply by Ken-JIAnomads** [reply]
Thanks it worked!
